pubspec.yaml

```diff
@@ -19,8 +19,8 @@ dependencies:
   mustache4dart: ^1.0.9
   package_config: '>=0.1.5 <2.0.0'
   path: ^1.3.0
-  quiver: '>=0.18.0 <0.23.0'
-  resource: ^1.1.0
+  quiver: '>=0.18.0 <0.25.0'
+  resource: ^2.1.2
```

Comment:
Sorry. I bumped the min constraint because

1) This is mostly used via the SDK, so I'm not too worried about folks getting a constraint issue here
2) Future updates would have to be careful about not using things that changed/broken w/ the v2 - branch. Trying to avoid the potential blowup...